lib/spack/spack/cmd/load.py

```diff
@@ -53,14 +53,29 @@ def setup_parser(subparser):
 the dependencies"""
     )
 
+    subparser.add_argument(
+        '--list',
+        action='store_true',
+        default=False,
+        help="show loaded packages: same as `spack find --loaded`"
+    )
+
 
 def load(parser, args):
     env = ev.active_environment()
+
+    if args.list:
+        results = spack.cmd.find.filter_loaded_specs(args.specs())
+        if sys.stdout.isatty():
+            spack.cmd.find.print_how_many_pkgs(results, "loaded")
+        spack.cmd.display_specs(results)
```

Comment:
My understanding is this cross-command re-use is frowned upon in Spack.  Instead, the `filter_loaded_specs` and `print_how_many_packages` should be moved out of the commands to a suitable (core) module.